projects/demo/src/modules/directives/dropdown-hover/examples/2/index.html

```diff
@@ -55,7 +55,9 @@
         [tuiDropdown]="settings"
         [(tuiDropdownOpen)]="openSettings"
         (tui-tab-activate.stop)="(0)"
-    ></button>
+    >
+        Open settings
```

Comment:
It didn't have text of purpose, let's use title instead. Also this should have caused a screen shot test to fail, why it didn't?

Reply:
fixed